examples/esp-idf-component/custom_partition.csv

```diff
@@ -0,0 +1,9 @@
+#
+# Custom partition format for two OTA partions
```

Comment:
This partition table currently has only one OTA partition, correct?

Reply:
no the comment in the file is out of place :P